src/components/SpacerView.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Animated, {useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
+import PropTypes from 'prop-types';
+import * as StyleUtils from '../styles/StyleUtils';
+import CONST from '../CONST';
+
+const propTypes = {
+    /**
+     * Should we show the spacer
+     */
+    shouldShow: PropTypes.bool.isRequired,
+
+    /**
+     * Array of style objects
+     * @default []
+     */
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```

Comment:
One last thing - please use already defined style propType inside `stylePropTypes.js`

Reply:
Sure, this is updated now. Thanks.

Reply:
Can you also remove the redundant eslint disable comment? It's the last thing bothering me :P

Reply:
@robertKozik done :)